docs/changelog/108038.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 108038
+summary: Excluding system data streams from global and factory retention
+area: Data streams
+type: enhancement
```

Comment:
Mary labeled all her PRs as `>non-issue` (and thus didn't include her changes in the changelog). I think it makes sense to do that here as well, for consistency's sake.

Reply:
Hmm I hadn't noticed that. I think I have to switch this to non-issue then since it would be odd to have an enhancement of a non-issue. :)